lib/provisioning/account.js

```diff
@@ -0,0 +1,292 @@
+const call_account_api = require('../network/call_account_api');
+
+/**
+ * @desc - List all sub_accounts under the account.
```

Comment:
Description should be in 3rd person, `Lists all sub accounts...` (mind `_` as well)

Reply:
Fixed